config/credentials/migration.yml.enc

```diff
@@ -0,0 +1 @@
+VheEW39+12tw/s5Md/RSstMteEt9mv+w+Ez7atRwY06woq8HiBSrQTt8gC0xziTwl12wEwUAUX9N6Ew1IZJIeI4GDoo=--PaiDlYsKLXjojRC4--sxJ5tDIDFDS5rrg1aW0VpQ==
```

Comment:
Can we use keyvault instead?

Reply:
We do for all new things, but some of the older variables in production are in the credentials file so the app won't boot unless it's present.

For this env it's there but empty.